Sources/Prototyper/Login/LoginView.swift

```diff
@@ -122,15 +132,15 @@ struct LoginView: View {
         if continueWithoutLogin {
             UserDefaults.standard.set(userid, forKey: UserDefaultKeys.username)
             state.continueWithoutLogin = true
-            self.finishLoggingIn = true
-            self.presentationMode.wrappedValue.dismiss()
         } else {
             apiHandler.login(userid,
                              password: password,
-                             success: { self.presentationMode.wrappedValue.dismiss()
-            }, failure: { _ in
-                self.showingLoginErrorAlert()
-            })
+                             success: {
```

Comment:
Shouldn't the success e.g. store the credentials or do something on success?

Reply:
I don't think so because the success call was used to dismiss the sheet and this is not longer necessary. I could update the method call without a success parameter 👍